### ISSUE TITLE
Progress loading screen gets stuck and never finishes

### ISSUE THREAD

**Issue by grymmy** [hackclub/sinerider#475]
### Description

For quite a while I have seen the loading screen occasionally get stuck at a certain percentage and never complete loading.  I experienced this again tonight, the screenshot below is after waiting 3 minutes.  I seriously do not think this is a fluke, it is a legitimate issue, it's extremely friction-inducing.

### Screenshots

<img width="1760" alt="Screenshot 2023-05-02 at 8 28 48 PM" src="https://user-images.githubusercontent.com/43505167/235812698-9c45684f-eb13-497a-b8b4-e38f17fae5a6.png">

**Comment by Benjamin-Rosenfeld** [comment]
This one has annoyed me for a while, maybe useful to not: reloading fixes it

**Comment by grymmy** [comment]
Yes - reloading does - but I would argue this is a very high priority bug as it induces friction prior to initial play

**Comment by Benjamin-Rosenfeld** [comment]
yes, I completely agree

**Comment by alhardwarehyde** [comment]
Logging Possibly an HTTPS issue

**Comment by karamassie** [comment]
@grymmy do you recall if you get it only via the daily puzzle flow or any time?

**Comment by maxwofford** [comment]
Happy to take this on– my approach would first be to add error logging on the loader, then fix whatever is getting thrown. Do we have any existing error reporting system?
